### ISSUE TITLE
Option to disable items translation

### ISSUE THREAD

**Issue by FVesely** [nette/forms#211]
I think it would be nice to have option to disable items translation for RadioList, CheckboxList, MultiSelectBox and SelectBox.

**Comment by dg** [comment]
use setTranslator(null)

**Comment by FVesely** [comment]
I know it could be done by setTranslator(null), but then you must translate caption by yourself. 

I use factories to create forms and then I set translator in Presenter. I think disabling items translation in form factory is nicer than this 
```php
/** @var \Nette\Forms\Controls\SelectBox $foo */
$foo = $form['foo'];
$foo->setTranslator();
$foo->caption = $trnalator->translate('foo');
```

**Comment by dg** [comment]
> but then you must translate caption by yourself.

This is solved in 3.0 #58

**Comment by FVesely** [comment]
Nice, thank you and sorry for this issue.
